src/FunctionApps/DSTP/src/main/kotlin/gov/cdc/prime/phdi/InjestHL7.kt

```diff
@@ -0,0 +1,39 @@
+package gov.cdc.prime.phdi
+
+import java.util.*
+import com.microsoft.azure.functions.*
+import com.microsoft.azure.functions.annotation.*
+import java.nio.charset.StandardCharsets
+
+class HL7Ingest {
+    @FunctionName("injesthl7")
+    fun run(
+        @HttpTrigger(
+            name = "req",
+            methods = [HttpMethod.GET, HttpMethod.POST],
+            authLevel = AuthorizationLevel.FUNCTION
+        ) request: HttpRequestMessage<Optional<String>>,
```

Comment:
This method isn't included in the PR anymore. This was originally in there as a means to mock your decrypt function, but given the decrypt function, I've removed it to avoid confusion. This applies to your comment below as well.